pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -231,8 +231,12 @@ func (pl *CSILimits) checkAttachableInlineVolume(vol v1.Volume, csiNode *storage
 		return fmt.Errorf("looking up provisioner name for volume %v: %w", vol, err)
 	}
 	if !isCSIMigrationOn(csiNode, inTreeProvisionerName) {
+		csiNodeName := ""
```

Comment:
Can a unit test with nil CSINode be added?

Reply:
There is no existing test set up but we can add a new TestCheckAttachableInlineVolume

Reply:
I think the existing test may need some tweaks to be able to pass in a nil CSIDriver:
https://github.com/kubernetes/kubernetes/blob/818dfc6f4147d36522dd4bf18146efec4832f1c1/pkg/scheduler/framework/plugins/nodevolumelimits/csi_test.go#L483

Reply:
Modified the test and now if it is a nil node the unit test throws the ```node not found```.

Reply:
Actually I needed to modify the getFakeCSINodeLister to fake returning a nil csi node.

Reply:
I guess this change is not necessary if we return the error?